vmb/matrimony/models/profiles.py

```diff
@@ -189,7 +189,10 @@ def send_batch_matches_email(self):
             category="DMD",
             status="NEW",
         )
-        email_message.send()
+        if email_message.sent_at == datetime.datetime.now().date():
```

Comment:
This conditional is in a bad location. You need to check if `EmailMessage` exists before creating it. If it exists, then return from there.